docs/TestPackage.php

```diff
@@ -0,0 +1,30 @@
+<?php
```

Comment:
Same here. If we need to highlight something, we should point to a concrete implementation in the existing code, otherwise we are repeating ourselves.